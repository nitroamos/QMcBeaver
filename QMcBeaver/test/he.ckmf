# This will test the Cambridge style Jastrows

&flags
# Parameters for QMC
run_type
 variational
dt
 0.01
number_of_walkers
 10
max_time_steps
 500
max_time
 -1
one_e_per_iter
 0
desired_convergence
 0
warn_verbosity
 0
rel_cutoff
 100
limit_branching
 1

# Parameters specific to the Green's function
sampling_method
 umrigar93_importance_sampling
QF_modification_type
 umrigar93_unequalelectrons
umrigar93_equalelectrons_parameter
 0.5
energy_modification_type
 umrigar93
energy_cutoff_type
 umrigar93
lock_trial_energy
 0
synchronize_dmc_ensemble
 0
synchronize_dmc_ensemble_interval
 1000

# Parameters specific to weights, branching, and fusion
walker_reweighting_method
 umrigar93_probability_weighted
branching_method
 nonunit_weight_branching
branching_threshold
 2
fusion_threshold
 0.45
population_control_parameter
 1
correct_population_size_bias
 1
old_walker_acceptance_parameter
 30

# Parameters for initialization
dt_equilibration
 0.01
use_equilibration_array
 0
equilibration_steps
 100
equilibration_function
 ramp
CKAnnealingEquilibration1_parameter
 500
walker_initialization_method
 dans_walker_initialization
walker_initialization_combinations
 3

# Parameters for added functionality/improvements
calculate_bf_density
 0
use_hf_potential
 0
hf_num_average
 100
replace_electron_nucleus_cusps
 1
nuclear_derivatives
 none
future_walking
 0

# Parameters relating to output
output_interval
 10
checkpoint
 0
checkpoint_interval
 100000
use_available_checkpoints
 0
checkpoint_input_name
 he_1.01
zero_out_checkpoint_statistics
 1
checkpoint_energy_only
 0
print_configs
 0
print_config_frequency
 50
temp_dir
 /temp1/amosa/he_1.01
write_all_energies_out
 0
write_electron_densities
 0
max_pair_distance
 -1
print_transient_properties
 0
print_transient_properties_interval
 10000
writePllxCorrelationDiagram
 0
pllxCorrelationDiagramMin
 0
pllxCorrelationDiagramMax
 0
writePllyCorrelationDiagram
 0
pllyCorrelationDiagramMin
 0
pllyCorrelationDiagramMax
 0
writePllzCorrelationDiagram
 0
pllzCorrelationDiagramMin
 0
pllzCorrelationDiagramMax
 0
writeOppxCorrelationDiagram
 0
oppxCorrelationDiagramMin
 0
oppxCorrelationDiagramMax
 0
writeOppyCorrelationDiagram
 0
oppyCorrelationDiagramMin
 0
oppyCorrelationDiagramMax
 0
writeOppzCorrelationDiagram
 0
oppzCorrelationDiagramMin
 0
oppzCorrelationDiagramMax
 0

# Parameters for wavefunction optimization
optimize_Psi
 0
optimize_EE_Jastrows
 1
optimize_EN_Jastrows
 1
optimize_NEE_Jastrows
 1
optimize_L
 1
optimize_CI
 0
optimize_Orbitals
 0
optimize_Psi_method
 automatic
optimize_Psi_criteria
 generalized_eigenvector
a_diag
 -1e-05
ksi
 0.5
max_optimize_Psi_steps
 3
equilibrate_first_opt_step
 1
equilibrate_every_opt_step
 1
optimization_max_iterations
 1
optimization_error_tolerance
 0.001
singularity_penalty_function_parameter
 1e-06
optimize_Psi_barrier_parameter
 1
numerical_derivative_surface
 umrigar88
line_search_step_length
 Linearize
ck_genetic_algorithm_1_population_size
 1000
ck_genetic_algorithm_1_mutation_rate
 0.2
ck_genetic_algorithm_1_initial_distribution_deviation
 1

# Parameters for computation/MPI
iseed
 -1000
parallelization_method
 manager_worker
mpireduce_interval
 100
mpipoll_interval
 5
walkers_per_pass
 1
use_basis_function_interpolation
 0
number_basis_function_interpolation_grid_points
 1000
basis_function_interpolation_first_point
 1e-10

# Parameters for the molecule and wavefunction
atoms
 1
charge
 0
energy
 -2.8616799749
trial_function_type
 restricted
norbitals
 1
nbasisfunc
 17
ndeterminants
 1
link_Jastrow_parameters
 1
link_NEE_Jastrows
 2
link_Orbital_parameters
 1
link_Determinant_parameters
 1
use_three_body_jastrow
 1
reproduce_NE_with_NEE_jastrow
 1
reproduce_EE_with_NEE_jastrow
 1

# Other parameters
chip_and_mike_are_cool
 Yea_Baby!
& 
&geometry
He   2      0                     0                     0                   
&
&basis
He    17    1
  1 s    
           293113.11    8978.14313323       
  1 s    
           55231.274    2567.72723787       
  1 s    
           12413.215    838.152993369       
  1 s    
           3284.5401    309.2188912         
  1 s    
           1006.2242    127.330123629       
  1 s    
           349.90692    57.6599804542       
  1 s    
           135.15749    28.2514134325       
  1 s    
           56.730644    14.7323957793       
  1 s    
           25.346385    8.05094968908       
  1 s    
           11.838633    4.54868916597       
  1 s    
           5.6964472    2.62792610722       
  1 s    
           2.7925021    1.53959050314       
  1 s    
           1.3837553    0.909294502104      
  1 s    
           0.6895619    0.539311940093      
  1 s    
           0.3445152    0.320491402051      
  1 s    
           0.1722927    0.190594449916      
  1 s    
           0.0861868    0.11336804552       
&
&wavefunction
  3.09931002e-07             1.94510305e-06             1.14430077e-05           
  5.34350301e-05             0.000208867627             0.000698890085           
  0.00206869828              0.00556002902              0.0138668294             
  0.0322104915               0.0688076682               0.130285337              
  0.211214257                0.28031275                 0.271806473              
  0.137337722                0.0192779849             

Alpha Occupation
1   

Beta Occupation
1   

CI Coeffs
  1                        

&

&Jastrow
ParticleTypes:	Electron_up	Electron_down
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.133479527303	0.506087144765	5.67378124145	-26.6370870631	72.8428191755	-79.8726111804	-23.7831278978	15.1448887985	28.955374462
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	2
Constants:	0.5	3	0

ParticleTypes:	Electron_up	He
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.373882098502	-0.582763472254	-1.8671331485	-4.44048044978	25.527785436	-61.6196485413	21.1285381046	78.9002523044	-76.663760563
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	2
Constants:	0	3	0

ParticleTypes:	He	Electron_up	Electron_down
threeBodyCorrelationFunctionType:	Cambridge
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:  4  4
Parameters:
 -0.490138585038      

 -1.47041575511        -1.47041575511        -4.41124726534         0                   
  0                     0                     25.5323715574       

  0.691755449355        0.691755449355       -27.1350808198        -27.1350808198       
  64.1404911938        -12.7661857787        -12.7661857787         11.2890662899       
  11.2890662899        -14.5197016023        -1.69385573205        -1.40359158575       
 -1.40359158575        -15.1895608836         18.3502185144         18.3502185144       
  0.739624849153        0.739624849153        35.7797243921       

  10.8218617576         10.8218617576         12.4981146899         12.4981146899       
 -78.8934229587        -78.8934229587         8.27662887604        -11.2890662899       
 -11.2890662899         7.25985080113         7.25985080113         0                   
  0                     0                     10.0168547335         10.0168547335       
 -46.1535820925        -46.1535820925         59.7101048913         59.7101048913       
  26.4990558826         5.33739192849        -2.62919113128        -2.62919113128       
  5.29640762956        -24.5699542109        -24.5699542109         2.49428366017       
  2.49428366017         9.23113147841         17.405338634          17.405338634        
  52.2160159019         52.2160159019         14.7310683076         14.7310683076       
  6.34406908499       

C:  3
Cutoff:  0.303546287567


&Jastrow


