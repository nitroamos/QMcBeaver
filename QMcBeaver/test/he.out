QMcBeaver version 20081114 was compiled in double precision.
Compiled on Nov 14 2008 18:47:55
Compiler version 3.4.6 20060404 (Red Hat 3.4.6-8)
With the libraries: BLAS LAPACK
Signaling: CHANNEL1 = 23 CHANNEL2 = 40


Laziness filter: 4
Fast growth:     5
Bad Energy:      100
Too heavy:       4

EupEdn:
Cambridge 2 particle jastrow parameters:
  g = 0.5  C = 3  L = 0.13348 (L not optimized)
x = r / 7.491786
(x - 1)^3 (0.5060871 -2.227631 x +5.673781 x^2 -26.63709 x^3 +72.84282 x^4 -79.87261 x^5 -23.78313 x^6 +15.14489 x^7 +28.95537 x^8 )

EupNuclear(He):
Cambridge 2 particle jastrow parameters:
  g = 0  C = 3  L = 0.3738821 (L not optimized)
x = r / 2.67464
(x - 1)^3 (-0.5827635 -1.74829 x -1.867133 x^2 -4.44048 x^3 +25.52779 x^4 -61.61965 x^5 +21.12854 x^6 +78.90025 x^7 -76.66376 x^8 )

EENuclear(He):
 Total Parameters = 64
 Free Parameters = 27
Cambridge 3 particle jastrow parameters:
  Nen = 4  Nee = 4  C = 3  L = 0.3035463 (optimized)
x1  = r1  / 3.29439
x2  = r2  / 3.29439
x12 = r12 / 3.29439
(x1 - 1)^3 (x2 - 1)^3 (
-0.4901386                               -1.470416  (x1 + x2)                     -4.411247  x1 x2                         +25.53237  x1 x2 x12                     
+0.6917554 (x1^2 + x2^2)                 -27.13508  (x1^2 x2 + x2^2 x1)           +64.14049  x1^2 x2^2                     -12.76619  (x1^2 + x2^2) x12             
+11.28907  (x1^2 x2 + x2^2 x1) x12       -14.5197   x1^2 x2^2 x12                 -1.693856  x12^2                         -1.403592  (x1 + x2) x12^2               
-15.18956  x1 x2 x12^2                   +18.35022  (x1^2 + x2^2) x12^2           +0.7396248 (x1^2 x2 + x2^2 x1) x12^2     +35.77972  x1^2 x2^2 x12^2               
+10.82186  (x1^3 + x2^3)                 +12.49811  (x1^3 x2 + x2^3 x1)           -78.89342  (x1^3 x2^2 + x2^3 x1^2)       +8.276629  x1^3 x2^3                     
-11.28907  (x1^3 + x2^3) x12             +7.259851  (x1^3 x2 + x2^3 x1) x12       +10.01685  (x1^3 + x2^3) x12^2           -46.15358  (x1^3 x2 + x2^3 x1) x12^2     
+59.7101   (x1^3 x2^2 + x2^3 x1^2) x12^2 +26.49906  x1^3 x2^3 x12^2               +5.337392  x12^3                         -2.629191  (x1 + x2) x12^3               
+5.296408  x1 x2 x12^3                   -24.56995  (x1^2 + x2^2) x12^3           +2.494284  (x1^2 x2 + x2^2 x1) x12^3     +9.231131  x1^2 x2^2 x12^3               
+17.40534  (x1^3 + x2^3) x12^3           +52.21602  (x1^3 x2 + x2^3 x1) x12^3     +14.73107  (x1^3 x2^2 + x2^3 x1^2) x12^3 +6.344069  x1^3 x2^3 x12^3               )

***************  TheMan.run();
 iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
       -90    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -80    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -70    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -60    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -50    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -40    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -30    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -20    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -10    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
         0   -2.9097385123   0.0000000e+00              10   -2.0000000000         0.98717         0.06512    0.0000745678        -2.90974 
       110   -2.9036594904   0.0000000e+00              10   -2.0000000000         0.99531         0.06233    0.0000391953        -2.90366 
       120   -2.9041178569   0.0000000e+00              20   -2.0000000000         0.99453         0.06033    0.0000603082        -2.90412 
       130   -2.9076604578   0.0000000e+00              30   -2.0000000000         0.99520         0.06214    0.0000741317        -2.90766 
       140   -2.9066671333   0.0000000e+00              40   -2.0000000000         0.99511         0.06177    0.0001345480        -2.90667 
       150   -2.9061927288   0.0000000e+00              50   -2.0000000000         0.99467         0.06179    0.0001166319        -2.90619 
       160   -2.9082411627   0.0000000e+00              60   -2.0000000000         0.99493         0.06164    0.0001258721        -2.90824 
       170   -2.9092007980   0.0000000e+00              70   -2.0000000000         0.99445         0.06137    0.0001281330        -2.90920 
       180   -2.9085742255   0.0000000e+00              80   -2.0000000000         0.99465         0.06101    0.0001195116        -2.90857 
       190   -2.9084016666   0.0000000e+00              90   -2.0000000000         0.99357         0.06086    0.0001126596        -2.90840 
       200   -2.9086776009   1.8777055e-03             100    3.1148452486      0.9932(11)      0.0610(10)    0.0001120608     -2.9087(19) 
       210   -2.9086589225   1.7732536e-03             110    3.2191743805      0.9933(11)     0.06065(96)    0.0001064691     -2.9087(18) 
       220   -2.9078465738               0             120   -1.0000000000     0.99350(94)     0.06090(94)    0.0001068215        -2.90785 
       230   -2.9072740480               0             130   -1.0000000000     0.99370(93)     0.06077(92)    0.0001081034        -2.90727 
       240   -2.9075420153               0             140   -1.0000000000     0.99356(85)     0.06035(89)    0.0001024632        -2.90754 
       250   -2.9074036501               0             150   -1.0000000000     0.99340(81)     0.06041(88)    0.0000989374        -2.90740 
       260   -2.9074398701   1.8855718e-03             160    5.9321158693     0.99356(76)     0.06040(86)    0.0000952958     -2.9074(19) 
       270   -2.9074882791   1.8855718e-03             170    6.1376599972     0.99357(73)     0.06042(84)    0.0000978971     -2.9075(19) 
       280   -2.9073666439   1.7062781e-03             180    5.3383130388     0.99374(69)     0.06029(81)    0.0000976222     -2.9074(17) 
       290   -2.9073870269   1.7062781e-03             190    5.7572478333     0.99351(68)     0.06019(80)    0.0000955755     -2.9074(17) 
       300   -2.9073794643   1.5617705e-03             200    5.3297943305     0.99351(65)     0.06022(79)    0.0000910704     -2.9074(16) 
       310   -2.9071875154   1.4433925e-03             210    4.8732116899     0.99359(65)     0.06009(76)    0.0000893511     -2.9072(14) 
       320   -2.9070094347   1.4433925e-03             220    5.2675091567     0.99362(60)     0.06024(75)    0.0000866179     -2.9070(14) 
       330   -2.9069236196   1.3697965e-03             230    5.0971291657     0.99373(60)     0.06014(72)    0.0000842989     -2.9069(14) 
       340   -2.9071388915   1.2914560e-03             240    4.8371137818     0.99365(56)     0.06002(71)    0.0000824083     -2.9071(13) 
       350   -2.9070037986   1.2914560e-03             250    5.1895097926     0.99369(54)     0.05976(69)    0.0000800262     -2.9070(13) 
       360   -2.9070386365   1.2159590e-03             260    4.9028199835     0.99378(52)     0.05983(68)    0.0000781073     -2.9070(12) 
       370   -2.9070890768   1.2159590e-03             270    5.1772616157     0.99365(53)     0.05971(66)    0.0000768228     -2.9071(12) 
       380   -2.9075283559   1.1561593e-03             280    4.6361460848     0.99344(56)     0.05960(65)    0.0000804419     -2.9075(12) 
       390   -2.9073147298   1.0975041e-03             290    4.3002518855     0.99338(55)     0.05952(63)    0.0000809499     -2.9073(11) 
       400   -2.9073458262   1.0975041e-03             300    4.5081096238     0.99351(54)     0.05926(62)    0.0000798894     -2.9073(11) 
       410   -2.9071761847   1.0382373e-03             310    4.2454789556     0.99360(53)     0.05927(61)    0.0000784558     -2.9072(10) 
       420   -2.9069358705   1.0476295e-03             320    4.3807130715     0.99371(51)     0.05929(60)    0.0000799211     -2.9069(10) 
       430   -2.9067756784   1.0476295e-03             330    4.5628191025     0.99366(51)     0.05939(59)    0.0000791367     -2.9068(10) 
       440   -2.9067092389   1.0248461e-03             340    4.5769853625     0.99369(49)     0.05948(58)    0.0000777924     -2.9067(10) 
       450   -2.9067995666   1.0248461e-03             350    4.7020824578     0.99377(48)     0.05937(57)    0.0000779565     -2.9068(10) 
       460   -2.9068953680   9.8444189e-04             360    4.5326097458     0.99373(47)     0.05928(57)    0.0000767585    -2.90690(98) 
       470   -2.9066145997   9.4781965e-04             370    4.1989453634     0.99374(46)     0.05941(56)    0.0000789474    -2.90661(95) 
       480   -2.9062525232   9.1452435e-04             380    3.7907171739     0.99359(46)     0.05951(55)    0.0000836197    -2.90625(91) 
       490   -2.9061573475   1.0351476e-03             390    5.0032534941     0.99359(45)     0.05952(54)    0.0000833109     -2.9062(10) 
       500   -2.9061530172   9.9407666e-04             400    4.7798784709     0.99360(44)     0.05961(54)    0.0000824890    -2.90615(99) 
       500   -2.9061530172   9.9407666e-04             400    4.7798784709     0.99360(44)     0.05961(54)    0.0000824890    -2.90615(99) 
***************  Print Results (ok = 1)
**************** Start of Results *****************

------------------- Energy -------------------
     -2.906153017175 +/- 0.000994076663       (400 samples, Tcorr    4.77988)
  DeCorr_depth       samples           Ave           Std        StdStd           Var        VarStd    Corr. Len.      Variance          Skew      Kurtosis
0                        400     -2.906153  4.546859e-04  1.609571e-05  2.067392e-07  1.463698e-08      0.000000      0.000082      0.126246      0.400621
1                        200     -2.906153  5.937431e-04  2.976165e-05  3.525309e-07  3.534155e-08      1.705196      0.000070      0.117589      0.393003
2                        100     -2.906153  7.393639e-04  5.254430e-05  5.466590e-07  7.769872e-08      2.644196      0.000054      0.119113      0.736656
3                         50     -2.906153  8.740746e-04  8.829487e-05  7.640064e-07  1.543526e-07      3.695507      0.000037     -0.224695      1.095712
             4            25     -2.906153  9.940767e-04  1.434826e-04  9.881884e-07  2.852654e-07      4.779878      0.000024     -0.105783      0.808184
5                         12     -2.906202  8.783800e-04  1.872712e-04  7.715514e-07  3.289906e-07      3.732003      0.000008      0.675635      0.123101
6                          6     -2.906202  9.383509e-04  2.967326e-04  8.805024e-07  5.568786e-07      4.259000      0.000004      0.315859     -0.592190
7                          3     -2.906202  8.128032e-04  4.064016e-04  6.606491e-07  6.606491e-07      3.195567      0.000001      0.000000      0.000000
8                          1     -2.907000

----------------- Energy^2 -------------------
      8.445807848188 +/- 0.005777846505       (400 samples, Tcorr    4.78169)

--------------- Kinetic Energy ---------------
      2.783077265704 +/- 0.229820126302       (400 samples, Tcorr   12.99208)

-------------- Potential Energy --------------
     -5.689230282878 +/- 0.229801715535       (400 samples, Tcorr   12.97621)

-------------- Nuc-Elec Energy ---------------
     -6.508194498803 +/- 0.228933994105       (400 samples, Tcorr   12.90788)

-------------- Elec-Elec Energy --------------
      0.818964215924 +/- 0.021374741654       (400 samples, Tcorr    7.09060)

------------ AcceptanceProbability -----------
      0.993599883328 +/- 0.000440002123       (400 samples, Tcorr    1.23160)

------------ DistanceMovedAccepted -----------
      0.059610860121 +/- 0.000535350612       (400 samples, Tcorr    0.00000)

------------- DistanceMovedTrial -------------
      0.060159544342 +/- 0.000542418392       (400 samples, Tcorr    0.00000)

----------------- Walker Age -----------------
      0.000000000000 +/- 1.000000000000e+300  (0 samples, Tcorr   -2.00000)
dt_effective: 0.00991 +/- 0.00126

--------------- Virial Ratio (-<V>/<T>) ---------------
      0: 2.04422 +/- 0.18792
         Total Energy Estimator (KE/Virial): -2.78308 +/- 0.22982
         Total Energy Estimator (PE/Virial): -2.84462 +/- 0.11490

**************** End of Results *****************
***************  Finalize
Average iterations per hour:                              8.9272e+06
Average iterations*walkers per hour:                      8.9272e+07
Average microseconds per sample:                          382.31
Average microseconds per sample per num initial walkers:  38.23

Wallclock Time:                          201 ms   =                0.201 s    =              0.00335 mins =           5.5833e-05 hrs 
Total Time:                              201 ms   =                0.201 s    =              0.00335 mins =           5.5833e-05 hrs     100.00 %
Initialization Time:                       1 ms   =                0.001 s    =           1.6667e-05 mins =           2.7778e-07 hrs       0.51 %
Equilibration Time:                       41 ms   =                0.041 s    =           0.00068333 mins =           1.1389e-05 hrs      20.52 %
Propagation Time:                        152 ms   =                0.152 s    =            0.0025333 mins =           4.2222e-05 hrs      75.84 %
Send Command Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Gather Properties Time:                    0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Synchronization Time:                      0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Poll for Command Time:                     0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Optimization Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %

      Evaluate basis functions: 0 ( avg =       2.94 us, n=           4995)      7.73 % cumulative =       7.73 %
         Matrix Multiplication: 0 ( avg =       2.04 us, n=           4995)      5.35 % cumulative =      13.08 %
             Derivative Ratios: 0 ( avg =       1.87 us, n=           4995)      4.91 % cumulative =      17.99 %
      Evaluate basis functions: 0 ( avg =       2.71 us, n=           4995)      7.10 % cumulative =      25.09 %
         Matrix Multiplication: 0 ( avg =       1.62 us, n=           4995)      4.24 % cumulative =      29.33 %
             Derivative Ratios: 0 ( avg =       0.88 us, n=           4995)      2.30 % cumulative =      31.63 %
              Potential Energy: 0 ( avg =       0.30 us, n=           4995)      0.78 % cumulative =      32.41 %
                    EN Jastrow: 0 ( avg =       1.72 us, n=           4995)      4.51 % cumulative =      36.92 %
                    EE Jastrow: 0 ( avg =       0.97 us, n=           4995)      2.55 % cumulative =      39.47 %
                   EEN Jastrow: 0 ( avg =       4.85 us, n=           4995)     12.74 % cumulative =      52.21 %
                Psi Quantities: 0 ( avg =       2.86 us, n=           4995)      7.51 % cumulative =      59.72 %
            Walker Propagation: 0 ( avg =       8.79 us, n=           4985)     23.07 % cumulative =      82.79 %
           Correlated Sampling: 0 ( avg =       0.00 us, n=            -15)      0.00 % cumulative =      82.79 %
         Calculate Observables: 0 ( avg =       2.72 us, n=           4985)      7.14 % cumulative =      89.93 %
                   Entire Step: 0 ( avg =      38.11 us, n=           4985)    100.00 % cumulative =     100.00 %
Wallclock Time:                          212 ms   =                0.212 s    =            0.0035333 mins =           5.8889e-05 hrs 
